### DISCUSSION TITLE
define a specific queue for all deliver_by methods

### DISCUSSION THREAD

**Discussion by junaidrasheed** [excid3/noticed#192] [Q&A] [answered]
Hi,

I was wondering if there is any way to specify a dedicate Queue to send all deliver_by workers to?. I am using Sidekiq for background job processing at the moment.

**Comment by excid3** [comment, marked as answer]
Define a job for it and tell Noticed to use the job. https://github.com/excid3/noticed/blob/master/lib/noticed.rb#L36

```ruby
Noticed.parent_class = "NotificationJob"
```

```ruby
class NotificationJob < ApplicationJob
  queue_as :notifications
end
```
